### ISSUE TITLE
intermittent timeout failures on travis-ci

### ISSUE THREAD

**Issue by itziakos** [simphony/simphony-jyulb#20]
When running a simulation as part of the tests can take more than 10 minutes in travis-ci which results in the build timing out. I am not sure what is the cause, but I see these failures every now and then

**Comment by kemattil** [comment]
Yes, I have seen this too. I'm not sure about the reason, but at least the original test cases took (on average) perhaps 6 min to complete (the wrapper test simulations are extremely! slow in travis-ci) ... due to this I made the test case much smaller (both for the file-io and internal wrapper): I was hoping that this would ease the problem.
